portlets/src/main/webapp/vue-app/realizations/components/Realizations.vue

```diff
@@ -29,7 +29,7 @@
             {{ getFromDate(props.item.createdDate) }}
           </td>
           <td class="text-truncate align-center wrap">
-            {{ props.item.creator && props.item.creator.remoteId || props.item.earner && props.item.earner.remoteId || '-' }}
+            {{ props.item.creator  || props.item.earner || '-' }}
```

Comment:
This will show the remote ID in the exported data ?

Reply:
yes changed in model from userInfo to string which takes only the remoteId